src/components/Form/input.story.vue

```diff
@@ -0,0 +1,199 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { Calendar, Search } from '@element-plus/icons-vue'
+const input = ref('')
+const input1 = ref('')
+const input2 = ref('')
+const input3 = ref('')
+const input4 = ref('')
+const textarea = ref('')
+const textarea1 = ref('')
+const textarea2 = ref('')
+const select = ref('')
+const text = ref('')
+</script>
+
+<template>
+  <Story title="Form/Input">
```

Comment:
with type: 'grid' you can specify suitable width
eg:
<Story title="From/Input" layout="{type: "grid", width: 500}">